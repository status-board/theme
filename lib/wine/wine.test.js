"use strict";

var _index = _interopRequireDefault(require("./index"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

test('Wine should match snapshot', function () {
  expect(_index["default"]).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy93aW5lL3dpbmUudGVzdC50cyJdLCJuYW1lcyI6WyJ0ZXN0IiwiZXhwZWN0Iiwid2luZSIsInRvTWF0Y2hTbmFwc2hvdCJdLCJtYXBwaW5ncyI6Ijs7QUFBQTs7OztBQUVBQSxJQUFJLENBQUMsNEJBQUQsRUFBK0IsWUFBWTtBQUMzQ0MsRUFBQUEsTUFBTSxDQUFDQyxpQkFBRCxDQUFOLENBQWFDLGVBQWI7QUFDSCxDQUZHLENBQUoiLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgd2luZSBmcm9tICcuL2luZGV4JztcblxudGVzdCgnV2luZSBzaG91bGQgbWF0Y2ggc25hcHNob3QnLCAoKTogdm9pZCA9PiB7XG4gICAgZXhwZWN0KHdpbmUpLnRvTWF0Y2hTbmFwc2hvdCgpO1xufSk7XG4iXX0=