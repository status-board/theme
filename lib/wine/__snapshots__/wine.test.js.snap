// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wine should match snapshot 1`] = `
Object {
  "charts": Object {
    "axis": Object {
      "legendColor": "hsl(6, 26%, 67%)",
      "textColor": "hsl(6, 26%, 67%)",
      "tickColor": "hsl(6, 26%, 67%)",
    },
    "colors": Array [
      "#a52818",
      "#e46a55",
      "#bd532e",
      "#9b2c13",
      "#E43D24",
    ],
    "grid": Object {
      "stroke": "#321515",
    },
    "tooltip": Object {
      "background": "#501919",
      "borderRadius": 0,
      "boxShadow": "0 2px 5px rgba(0, 0, 0, 0.6)",
      "color": "hsl(10, 60%, 90%)",
      "fontSize": "1.4vmin",
    },
  },
  "colors": Object {
    "background": "#281212",
    "failure": "#a31c12",
    "icon": "hsl(0, 52%, 60%)",
    "success": "#50a3b2",
    "text": "hsl(6, 26%, 67%)",
    "textHighlight": "#fff",
    "unknown": "#7e706d",
    "warning": "#b87334",
  },
  "dashboard": Object {
    "header": Object {
      "background": "rgb(69, 23, 23)",
    },
    "player": Object {
      "slash": Object {
        "color": "hsl(0, 52%, 60%)",
      },
    },
  },
  "label": Object {
    "addon": Object {
      "background": "rgb(55, 21, 21)",
      "color": "hsl(0, 60%, 63%)",
    },
    "extend": "
        border-radius: 2px;
        ",
    "main": Object {
      "background": "#561d1d",
      "color": "hsl(10, 60%, 90%)",
    },
  },
  "list": Object {
    "item": Object {
      "extend": "
            border-bottom: 1px solid rgb(52, 20, 20);
            &:last-child {
                border-bottom: 0;
            }
            ",
      "hover": Object {
        "background": "rgb(82, 25, 25)",
      },
      "meta": Object {
        "color": "hsl(0, 52%, 60%)",
        "fontSize": "1.6vmin",
      },
    },
  },
  "name": "wine",
  "notifications": Object {
    "item": Object {
      "background": "rgb(87, 25, 25)",
      "color": "#fff",
      "extend": "
            box-shadow: 0 1px 2px rgba(0, 0, 0, .75);
            font-size: 1.6vmin;
            ",
    },
  },
  "root": Object {
    "background": "#281212",
    "extend": "
        @import url(\\"https://fonts.googleapis.com/css?family=Roboto+Slab:100,300|Open+Sans:400italic,400,300,600,700\\");
        & a {
            color: hsl(10, 60%, 90%);
        }
        ",
  },
  "typography": Object {
    "default": Object {
      "default": Object {
        "fontFamily": "'Open sans', sans-serif",
        "fontSize": "1.8vmin",
        "fontWeight": 400,
        "lineHeight": "1.5em",
      },
      "small": Object {
        "fontSize": "1.6vmin",
      },
      "strong": Object {
        "fontWeight": 600,
      },
    },
    "display": Object {
      "default": Object {
        "fontFamily": "'Roboto Slab', sans-serif",
        "fontSize": "1.8vmin",
        "lineHeight": "1.5em",
      },
    },
    "mono": Object {
      "default": Object {
        "fontFamily": "'Space Mono', Consolas, monospace",
        "fontSize": "1.8vmin",
        "lineHeight": "1.4em",
      },
    },
  },
  "widget": Object {
    "background": "rgb(69, 23, 23)",
    "header": Object {
      "count": Object {
        "color": "hsl(0, 52%, 60%)",
        "extend": "
                background-color: rgb(40, 18, 18);
                border-radius: 2px;
                padding: 0.6vmin 1vmin;
                ",
        "fontSize": "1.8vmin",
      },
      "extend": "
            border-bottom: 1px solid rgb(40, 18, 18);
            ",
      "fontSize": "2.2vmin",
      "icon": Object {
        "color": "hsl(0, 52%, 60%)",
        "fontSize": "2.6vmin",
      },
      "subject": Object {},
    },
    "wrapper": Object {
      "padding": "0.3vmin",
    },
  },
}
`;
